src/openzeppelin/access/ownable.cairo

```diff
@@ -0,0 +1,54 @@
+#[contract]
+mod Ownable {
+    use starknet::ContractAddress;
+    use starknet::ContractAddressZeroable;
+    use starknet::contract_address_const;
+    use starknet::get_caller_address;
+    use zeroable::Zeroable;
+
+    struct Storage {
+        _owner: ContractAddress
+    }
+
+    #[event]
+    fn OwnershipTransferred(previous_owner: ContractAddress, new_owner: ContractAddress) {}
+
+    #[internal]
+    fn initializer() {
+        let caller: ContractAddress = get_caller_address();
+        _owner::write(caller);
+    }
+
+    #[internal]
+    fn assert_only_owner() {
+        let owner: ContractAddress = _owner::read();
+        let caller: ContractAddress = get_caller_address();
+        assert(!caller.is_zero(), 'Caller is the zero address');
+        assert(caller == owner, 'Caller is not the owner');
+    }
+
+    #[internal]
+    fn owner() -> ContractAddress {
+        _owner::read()
+    }
+
+    #[internal]
+    fn transfer_ownership(new_owner: ContractAddress) {
+        assert(!new_owner.is_zero(), 'New owner is zero address');
```

Comment:
```suggestion
        assert(!new_owner.is_zero(), 'New owner is the zero address');
```
closer to [OZ Contracts for solidity's error message](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/access/Ownable.sol#L70) (still lacking the `Ownable:` we can't add due max lenght)